src/styles/main.scss

```diff
@@ -1,3 +1,193 @@
-body {
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
+
+$color: #00acdc;
+$color-black: #060b35;
+
+* {
   margin: 0;
+  padding: 0;
```

Comment:
it's bad practice to reset `padding` or `margin` here (it might affect performance)
reset it for list of tags you really need

in `*` selector set only `box-sizing`